lingvo/core/beam_search_helper.py

```diff
@@ -88,10 +88,11 @@ def InitBeamSearchState(theta, encoder_outputs, num_hyps_per_beam):
         Returns:
           initial_results: a `.NestedMap` of initial results. It should contain
               the following tensors at the minimum.
-                  log_probs: The initial log probs for each of the tokens in
-                      the target vocab, of shape [tgt_batch, vocab_size].
-                  atten_probs: The initial attention probs, of shape [tgt_batch,
-                      src_len].
+          log_probs: The initial log probs for each of the tokens in
+              the target vocab, of shape [tgt_batch * num_hyps_per_beam, 
+              vocab_size].    
+          s: The initial attention probs, of shape [
```

Comment:
Where are you reading this from? it's not in the test

Reply:
from here: 

https://github.com/tensorflow/lingvo/blob/master/lingvo/core/beam_search_helper.py#L358
```
    num_hyps = tf.shape(initial_results.log_probs)[0]
    num_beams = num_hyps // num_hyps_per_beam
```